test-configs.yaml

```diff
@@ -1635,16 +1675,56 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6qp-sabresd
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sll-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sx-sdb
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6ul-14x14-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6ull-14x14-evk
+    test_plans:
+      - baseline
+      - kselftest
+
   - device_type: imx7d-sdb
     test_plans:
       - baseline
       - kselftest
 
+  - device_type: imx7ulp-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx8mm-ddr4-evk
```

Comment:
No baseline results on staging.

Device online in the lab:
https://lavalab.nxp.com/scheduler/device_type/imx8mm-ddr4-evk